src/classes/ERR_RecordError.cls

```diff
@@ -43,8 +43,8 @@ public class ERR_RecordError {
 
     private String context;
 
-    private Map<ID, Error__c> insRecordErrors = new Map<Id, Error__c>();
-    private Map<ID, Error__c> updRecordErrors = new Map<Id, Error__c>();
+    public Map<ID, Error__c> insRecordErrors = new Map<Id, Error__c>();
```

Comment:
if this change was for testing purposes maybe we can use @TestVisible flag. 
If for some reason there's any security concern we could also create get/set methods to avoid changing visibility

Reply:
@grivero This is necessary so I can extract the errors from the instance of the ALLO_AllocationService class when working with it in the BDI_GAUAllocationsUtil code.  There isn't a particular reason these needed to be private, and I ran it by John brandolini who originally wrote the code and he didn't see an issue with it.